client/lib/upgrades/actions/domain-management.js

```diff
@@ -290,6 +290,19 @@ function deleteDns( domainName, record, onComplete ) {
 	} );
 }
 
+function addDnsOffice( domainName, token, onComplete ) {
+	wpcom.addDnsOffice( domainName, token, ( error, data ) => {
+		if ( ! error ) {
+			Dispatcher.handleServerAction( {
+				type: ActionTypes.DNS_FETCH_COMPLETED,
```

Comment:
I know it's going to be the same thing, but reusing same action type in a different setting might create some confusion in the future. Maybe we follow pattern of the other action types: `DNS_ADD_OFFICE`, `DNS_ADD_OFFICE_COMPLETED`, `DNS_ADD_OFFICE_FAILED`. This way we can also handle cases at the beginning of the request and the errors.